examples/with-tailwindcss/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
I don't think we should include ESLint - while it is helpful, I think this example should be as simple as possible - just Next.js and Tailwind.